tests/testthat/test-DiseasystoreBase.R

```diff
@@ -9,9 +9,9 @@ test_that("DiseasystoreBase works", {
   expect_error(DiseasystoreBase$new(source_conn = "/some/path"), regexp = "target_conn option not defined")
 
   # 3)
-  options(diseasystore.source_conn = "/some/path")
+  options("diseasystore.source_conn" = "/some/path")
```

Comment:
R allows periods for list/arguments names, and these changes therefore do nothing. Reverting these changes further clutters the history, so I will allow this for now.

Is this covered by the Diseasy (or Tidyverse) style guide? Should we consider it?

Reply:
I don't think it is part of the style guides but it is, in my opinion more clear.
While R does allow names that are not explicitly quoted, R inconsistenly quotes the names of inputs (sometimes, not always)

Quoting names explicitly is a good habit that reduces the chance of errors.

`options(1 = 2)`    Fails
`options("1" = 2)` Works 
`options(_MY_BUILD_OPTION_ = FALSE)`      Fails
`options("_MY_BUILD_OPTION_" = FALSE)`   Works

Reply:
Well, to go off a pedantic tangent: Those examples aren't exclusive to `options` or any other functions, really, but comes down to the R language; variable names, and (i would assume) by extension parameter and list names must match `\.?[^0-9_][0-9A-z._]` and may not be reserved words (e.g. `TRUE`, `NULL`, `if`).

Both your examples violate this limit to variable names, therefore needing quotes.¹

In order to not make the comment thread of some pull request be a much larger philosophical space than it accommodates, I will repeat that I will allow this for now 🙂 Personally, I would prefer quoting only when necessary, and I am unsure if these mixed styles will impact readability.

<hr/>

¹ 🎉**BONUS ROUND**🎉: TIL argument names may also be quoted, regardless of necessity:
```r
seq_len("length.out" = 3)
#> [1] 1 2 3
```